doc/CMakeLists.txt

```diff
@@ -61,52 +43,72 @@ file(GLOB_RECURSE NEW_CHANGE_NOTES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} "etc/new
-    ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE)
+    execute_process(COMMAND ${CYGPATH} -u ${CMAKE_CURRENT_SOURCE_DIR}
+        WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
+        OUTPUT_VARIABLE DocSourcePath
+        ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+    execute_process(COMMAND ${CYGPATH} -u ${CMAKE_CURRENT_BINARY_DIR}
+        WORKING_DIRECTORY "${CMAKE_BINARY_DIR}"
+        OUTPUT_VARIABLE DocBinaryPath
+        ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE)
 else()
-  set(CMAKE_CURRENT_SOURCE_DIR_UNIX_FORMAT ${CMAKE_CURRENT_SOURCE_DIR})
-  set(CMAKE_CURRENT_BINARY_DIR_UNIX_FORMAT ${CMAKE_CURRENT_BINARY_DIR})
+    set(DocSourcePath ${CMAKE_CURRENT_SOURCE_DIR})
+    set(DocBinaryPath ${CMAKE_CURRENT_BINARY_DIR})
 endif()
 
 foreach(m_src ${MAN_PAGES})
-  # troff formatted man pages
-  set(m_target ${CMAKE_CURRENT_BINARY_DIR}/${m_src})
-  add_custom_command(OUTPUT ${m_target}
-#    COMMAND soelim -I${CMAKE_CURRENT_SOURCE_DIR}  ${m_src} | sed '/^[.]XX ./d' > ${m_target}
-    COMMAND soelim -I${CMAKE_CURRENT_SOURCE_DIR} ${m_src} > ${m_target}
-    DEPENDS ${m_src} ${MAN_COMMON}
-    COMMENT "Building ${m_target}"
-    VERBATIM)
-  list(APPEND MAN_PAGE_OUTPUTS ${m_target})
-
-  # HTML formatted man pages for web viewing
-  set(m_target ${CMAKE_CURRENT_BINARY_DIR}/html/man/${m_src}.html)
-  cmake_path(GET m_src STEM image_stem_name)
-  cmake_path(GET m_target PARENT_PATH m_target_path)
-  add_custom_command(OUTPUT ${m_target}
-    COMMAND ${GROFF} -Thtml -t -man -I${CMAKE_CURRENT_SOURCE_DIR_UNIX_FORMAT} -I${CMAKE_CURRENT_BINARY_DIR_UNIX_FORMAT} -P-I${image_stem_name} ${m_src} > ${m_target}
-    DEPENDS ${m_src} ${MAN_COMMON}
-    WORKING_DIRECTORY ${m_target_path}
-    COMMENT "Building ${m_target} in ${m_target_path}"
-    VERBATIM)
-  list(APPEND HTML_MAN_PAGE_OUTPUTS ${m_target})
+    # troff formatted man pages
+    set(m_target ${CMAKE_CURRENT_BINARY_DIR}/${m_src})
+    add_custom_command(OUTPUT ${m_target}
+        #COMMAND soelim -I${CMAKE_CURRENT_SOURCE_DIR}  ${m_src} | sed '/^[.]XX ./d' > ${m_target}
+        COMMAND soelim -I${CMAKE_CURRENT_SOURCE_DIR} ${m_src} > ${m_target}
+        DEPENDS ${m_src} ${MAN_COMMON}
+        COMMENT "Building ${m_target}"
+        VERBATIM)
+    list(APPEND MAN_PAGE_OUTPUTS ${m_target})
+
+    # HTML formatted man pages for web viewing
+    set(m_target ${CMAKE_CURRENT_BINARY_DIR}/html/man/${m_src}.html)
+    cmake_path(GET m_src STEM image_stem_name)
+    cmake_path(GET m_target PARENT_PATH m_target_path)
+    add_custom_command(OUTPUT ${m_target}
+        COMMAND
+            sh ${CMAKE_CURRENT_SOURCE_DIR}/script/groff.sh
+            -T html                # Output device
+            -man                   # Macro Package man
+            # Order of preprocessors may be important
+            -t                     # Preprocess with tbl
+            -s                     # Preprocess with soelim, with search paths:
+            -I ${DocSourcePath}
```

Comment:
Adding new and moving existing options shouldn't be mixed with style changes.

Reply:
Yes that's true. I tried to kept the numbers of commits low for Scott ;-)